Dockerfile

```
@@ -15,14 +15,13 @@ COPY . /app/
 ENTRYPOINT poetry run gunicorn --bind 0.0.0.0:5000 todo_app.wsgi:app
 
 FROM base as test
-RUN apt-get update && apt-get install curl -y
-RUN apt-get update && apt-get install wget -y
 #Install chrome
 RUN ls -lrt /etc/apt
-RUN apt-get update && apt-get install -y gnupg2
+RUN apt-get update && apt-get install curl wget gnupg2 -y
 RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
 RUN sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
 RUN apt-get update && apt-get install google-chrome-stable -y
 ENV PYTHONPATH /app:$PATH
+COPY . /app/
```

This will work on Travis, but not locally from a Windows box if you already have a .venv folder.

* You copy pyproject.toml in and then run `poetry install`, creating a virtual env in a cache folder somewhere
* You copy in the rest of the app, including your local .venv folder since it isn't .dockerignored
* When doing `poetry run pytest` poetry tries to use the `/app/.venv` folder and fails because it has Windows executables in rather than Linux ones

It works on Travis because there isn't a .venv folder already. One way to fix it would be to just copy in the `todo_app/` folder, since that's all you need anyway.

```suggestion
COPY ./todo_app /app/todo_app
```